dataprofiler/profilers/graph_profiler.py

```diff
@@ -422,21 +422,35 @@ def _get_continuous_distribution(
                         best_distrib = distribution
                         best_fit = distribution.name
                         best_mle = mle
-                        best_fit_properties = fit
-
-                mean, variance, skew, kurtosis = best_distrib.stats(
-                    best_fit_properties, moments="mvsk"
-                )
+                        best_fit_attributes = fit[:-2]
+                        best_fit_scale = fit[-1]
+                        best_fit_loc = fit[-2]
+                if best_fit_attributes:
+                    mean, variance, skew, kurtosis = (
```

Comment:
I think it would be better to iterate through expected keys here rather than the dict itself. 
What I mean is if the best_distrib.stats for some reason differs from the expected 4 values (mean, variance, skew, kurtosis), it could lead to an error. Now whether we want an error if that happens is up for debate. @taylorfturner what is the direction we want to go with this?

Reply:
Disregard! This makes use of an external library